unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/AvatarEditorHUDController.cs

```diff
@@ -11,31 +11,35 @@
 
 public class AvatarEditorHUDController : IHUD
 {
-    protected static readonly string[] categoriesThatMustHaveSelection = {Categories.BODY_SHAPE, Categories.UPPER_BODY, Categories.LOWER_BODY, Categories.FEET, Categories.EYES, Categories.EYEBROWS, Categories.MOUTH};
-    protected static readonly string[] categoriesToRandomize = {Categories.HAIR, Categories.EYES, Categories.EYEBROWS, Categories.MOUTH, Categories.FACIAL, Categories.HAIR, Categories.UPPER_BODY, Categories.LOWER_BODY, Categories.FEET};
+    private const int LOADING_OWNED_WEARABLES_RETRIES = 3;
+    private const string LOADING_OWNED_WEARABLES_ERROR_MESSAGE = "There was a problem loading your collectibles";
```

Comment:
This message look too generic for me, I think that we should give more information to the user so he can understand what failed. Something like:

"There was a problem loading your wearable collectibles for the avatar editor, if you want to change your wearables with your collectibles, please { insert action to do it}"

Reply:
Yeah, it's a temporal message. I'm currently confirming with @olavra the definitive messages for the user feedback.